homeassistant/components/deconz/config_flow.py

```diff
@@ -20,6 +20,7 @@
 from homeassistant import config_entries
 from homeassistant.components import ssdp
 from homeassistant.components.deconz.gateway import DeconzGateway
+from homeassistant.components.hassio.discovery import HassioServiceInfo
```

Comment:
Generally, when we import from other integrations we do so from the base module, why is that not done for discovery?

Reply:
Hassio and Mqtt generate the discovery flow from `discovery.py`, so that is where I created the corresponding classes.
The others (zeroconf/dhcp/usb/homekit/) generate the discovery flow from `init.py`.

I can move the class over to `init.py` (either in this PR or a follow-up PR) if the others agree.